File: find_digits/fd.py
def find_digits(n:int)->int:
    count = 0
    for digit in str(n):
        if int(digit) == 0:
            continue
        elif n % int(digit) == 0:
            count+=1
    return count

if __name__ == "__main__":
    print(find_digits(12))
    print(find_digits(1012))

File: find_digits/fd.js
const findDigits = (n) => {
	let count = 0;
	for(let i = 0; i< n.toString().length; i++){
		if(parseInt(n.toString()[i]) === 0){
			continue;
		} else if(n % parseInt(n.toString()[i]) === 0){
			count++;
		}
	}
	return count;
}

console.log(findDigits(12));
console.log(findDigits(1012));